### PR TITLE
Bundler 2.3 docs and Bundler 2.3 blog post draft

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
### What was the end-user problem that led to this PR?

We haven't published Bundler 2.3 docs, nor a release blog post.

### What is your fix for the problem, implemented in this PR?

Publish docs and write a blog post draft.

### CODE REVIEW

File: source/blog/2022-01-23-bundler-v2-3.html.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+date: 2022-01-23 16:07 UTC
+tags:
+author: David Rodríguez
+author_url: https://github.com/deivid-rodriguez/
+category: release
+---
+
+2021 saw a fair amount of development in the RubyGems & Bundler repositories. We
+tried to release more often than ever to catch and fix bugs and distribute our
+improvements as early as possible users. That has led to 33 patch level versions
```
Comment (olleolleolle) [inline source/blog/2022-01-23-bundler-v2-3.html.markdown:13]:
```suggestion
improvements as early as possible to our users. That has led to 33 patch level versions
```

File: source/blog/2022-01-23-bundler-v2-3.html.markdown [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+in the Bundler 2.x series released about a year ago.
+
+Our goal for Bundler 2.3 was to implement a long wanted feature of being able to
```
Comment (olleolleolle) [inline source/blog/2022-01-23-bundler-v2-3.html.markdown:16]:
```suggestion
Our goal for Bundler 2.3 was to implement a long-wanted feature of being able to
```

File: source/blog/2022-01-23-bundler-v2-3.html.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+fully control the version of Bundler itself an application runs. There's a long
+story with this feature, because it was shipped a few years ago in a manner that
+was too strict and ended up causing more harm than good, so had to be partially
+reverted.
+
+## So, how did things work before Bundler 2.3?
+
+Up until now, RubyGems would prefer to activate the version of Bundler recorded
+in the `Gemfile.lock` file if available, and would fallback to the highest
```
Comment (olleolleolle) [inline source/blog/2022-01-23-bundler-v2-3.html.markdown:25]:
```suggestion
in the `Gemfile.lock` file if available, and would fall back to the highest
```

**Comment by olleolleolle** [comment]
(sorry for drive-by nitpicking!)

**Comment by deivid-rodriguez** [comment]
Thank you @olleolleolle 😍

**Comment by indirect** [comment]
@deivid-rodriguez added some small tweaks, but this looks great!

**Comment by deivid-rodriguez** [comment]
Awesome, thank you! Much better wording now 😅.

**Comment by deivid-rodriguez** [comment]
I want to add a note about RubyGems 3.3 also being required for this feature to work as explained, but I'm not sure where to add this information. Any ideas?

**Review by indirect — APPROVED** [review]
I'm happy with this! Thanks for all your hard work, @deivid-rodriguez 😁

**Comment by deivid-rodriguez** [comment]
Thank you so much @indirect!

**Pull request merged**